libs/sitemaps/src/createDatabasePaginator.ts

```diff
@@ -2,10 +2,21 @@ import { models, ThreadInstance, UserAttributes } from '@hicommonwealth/model';
 import { getThreadUrl } from '@hicommonwealth/shared';
 import { Op } from 'sequelize';
 
+const THREAD_PRIORITY: number = parseFloat(
+  process.env.SITEMAP_THREAD_PRIORITY || '0.8',
+);
+const PROFILE_PRIORITY: number = parseFloat(
+  process.env.SITEMAP_PROFILE_PRIORITY || '0.4',
```

Comment:
Should create `libs/sitemaps/src/config.ts` and set these variables in there using the same pattern as other config files.

Reply:
+1

Reply:
ok.. fair enough.